code/game/objects/items.dm

```diff
@@ -632,3 +640,34 @@ GLOBAL_VAR_INIT(rpg_loot_items, FALSE)
 		M.dirty++
 
 /obj/item/proc/on_mob_death(mob/living/L, gibbed)
+
+/obj/item/proc/update_force_string()
+	if(force)
+		switch(force)
+			if(1 to 3)

```

Comment:
Nothing forces `force` to be an integer. Also this and the following case share `force==3`